getinsomnia/insomnia
# chore(pre-req): clean up pre-request script sdk objects - INS-3379

### Please review this PR (https://github.com/Kong/insomnia/pull/7171) firstly as current one is based on it

Changes:
- [x] enable `eval()`
- [x] enable requiring node.js modules
- [x] make `console.log` outputs more friendly
- [ ] enable requiring npm modules -- will do this in a separated PR
- [x] enable built-in lodash module
- [x] avoid exposing `fs` as it is for internal usage

### How to test
Run this script and check if it runs well:

```javascript
// eval the express and set it
const evalResult = eval('8+8');
insomnia.environment.set('evalResult', evalResult);
console.log({
  field1: {field2: 'boom'}
});
```
